Tasks/count.js

```diff
@@ -1,17 +1,12 @@
-// Sum all number values in dict
+'use strict'
```

Comment:
```suggestion
'use strict';
```

Reply:
use strict';

const adder = (obj) => {
let sum = 0;
const arr_of_values = Object.values(obj);
for (const element of arr_of_values) {
if (typeof element === 'number') sum += element;
}
return sum;
};

require('../Tests/count.js')(adder)